gravitee-apim-portal-webui-next/src/app/registration/registration.component.html

```diff
@@ -0,0 +1,118 @@
+                <mat-error i18n="@@registrationEmailErrorRequired">Email is required</mat-error>
+              }
+              @if (registrationForm.controls.email.hasError('email')) {
+                <mat-error i18n="@@registrationEamilErrorFormat">Email format error</mat-error>
+              }
+            </mat-form-field>
+
+            @if ((customUserFields ?? []).length > 0) {
+              <div formGroupName="customFields">
+                @for (customField of customUserFields; track customField.key) {
+                  @if ((customField.values ?? []).length === 0) {
+                    <mat-form-field appearance="outline" class="registration__form__input">
+                      <mat-label>{{ customField.label }}</mat-label>
+                      <input matInput [formControlName]="customField.key!" />
+                      @if (registrationForm.controls.lastname.hasError('required')) {
+                        <mat-error i18n="@@registrationUserFieldValueErrorRequired">Value is required</mat-error>
+                      }
+                    </mat-form-field>
+                  }
+                  @if ((customField.values ?? []).length > 0) {
+                    <mat-form-field appearance="outline">
+                      <mat-label>{{ customField.label }}</mat-label>
+                      <mat-select [formControlName]="customField.key!" [aria-label]="customField.label!" data-testId="grantTypes">
+                        @for (customFieldValue of customField.values; track customFieldValue) {
+                          <mat-option [value]="customFieldValue">{{ customFieldValue }}</mat-option>
+                        }
+                      </mat-select>
+                    </mat-form-field>
+                  }
+                }
+              </div>
+            }
+
+            @if (error() === 401) {
+              <mat-error class="registration__form__error m3-label-large" i18n="@@registrationError401">Unauthorized </mat-error>
+            }
+          </div>
+          <div class="registration__form__buttons">
+            <button
+              [disabled]="registrationForm.invalid"
+              type="submit"
+              mat-flat-button
+              i18n="@@registrationAction"
+              class="registration__form__submit">
+              SIGN UP
+            </button>
+            <a class="registration__form__forgot-password m3-label-large" i18n="@@registrationBackToPortalAction" routerLink="/log-in">
+              Back to log in
+            </a>
+          </div>
+        </div>
+      </mat-card-content>
+    </mat-card>
+  </form>
+}
+
+@if (submitted) {
+  <div class="registration__success__container">
+    <div class="registration__success__container__icon">
+      <mat-icon>outgoing_mail</mat-icon>
+    </div>
+
+    <h2 i18n="@@registrationSuccessHeader">Registration success</h2>
+    <div i18n="@@registrationSuccessLabel">An email has been sent to:</div>
+    <div>{{ sentToEmail }}</div>
```

Comment:
Related to https://github.com/gravitee-io/gravitee-api-management/pull/14707/changes#r2640097808

Retrieve value of a signal

```suggestion
    <div>{{ sentToEmail() }}</div>